### PR TITLE
Movie list

### PR DESCRIPTION
**Pull request by beetlejays**

### CODE REVIEW

File: src/components/Movie/Movie.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
```
Comment (isabellpaulmann) [inline src/components/Movie/Movie.js:1]: Schön, dass du endlich Styled Component benutzt :D

File: src/App.js [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
-          <div className="movie__single" key={movie.id}>
-            <h3 className="movie__title">{movie.title}</h3>
-          </div>
-        ))}
-      </div>
+      <main>
+        <div className="movie__container">
+          {error && (
+            <div>
+              <h4 style={{color: '#3083dc'}}>Ooooops... Something went wrong.</h4>
```
Comment (ahohnsen) [inline src/App.js:42]: If you do not have an `h1`-`h3` in your document, you should not use an `h4` here.

File: src/components/Header/Header.spec.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react';
+
+import Header from './Header';
+
+describe('Header', () => {
+  it('Shows that the header component is rendered', () => {
+    render(<Header />);
+    screen.getByAltText('mov.me logo');
+  });
```
Comment (ahohnsen) [inline src/components/Header/Header.spec.js:9]: You are missing something here to complete your test:

```suggestion
  it('renders header with app logo', () => {
    render(<Header />);
    const logo = screen.getByAltText('mov.me logo');
    
    expect(logo).toBeInTheDocument()
  });
```

File: src/components/Movie/Movie.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  height: 3rem;
+  border-radius: 50%;
+  background-color: #3083dc;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 2px 5px 5px grey;
+`;
+
+const MovieTitle = styled.h3`
```
Comment (ahohnsen) [inline src/components/Movie/Movie.js:46]: Same here as mentioned beforehand: Your headlines should be chronologically - meaning you should only use a `h3` if there is already an `h1` and `h2`.

File: src/components/Movie/Movie.spec.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+describe('Movie', () => {
+  it('Shows that the movie poster is being renderd', () => {
+    render(<Movie />);
+    screen.getByRole('img');
+  });
+  it('Shows that the movie title is being renderd', () => {
+    render(<Movie />);
+    screen.getByRole('heading');
+  });
+});
```
Comment (ahohnsen) [inline src/components/Movie/Movie.spec.js:14]: Same as in your previous test - please have a look at my comment in your other test file.

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Hey Jay, 

good job 💪🏼 I made some suggestions here and there. Please have a look :)

File: src/components/Header/Header.spec.js (hunk 3 above)
Comment (beetlejays) [reply src/components/Header/Header.spec.js:9]: This was pretty hidden in the documentation... :)
https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument'

**Pull request closed without merging**